### DISCUSSION TITLE
Implementation Guide

### DISCUSSION THREAD

**Discussion by Derpitron** [AlexvZyl/nordic.nvim#86] [General]
Using Aurora colors in Nord for more contrast looks like a great idea, by your screenshots!

I would appreciate if you drafted a generalized color palette/list of colors, which I and other people could use to develop themes for other applications like VSCode or terminals.

**Comment by AlexvZyl** [comment]
Glad you like it!

That is definitely an option, however it will mostly depend on how popular the theme gets.  I do not think it is there yet.  Just VSCode and a terminal request here and there #18.

Just some points worth mentioning:

- I am an E&E engineer, I actually do not know a ton about color theory and design.  I just picked colors that I liked.
- Once the theme is actually mature I am going to add a **full** description of the color palette.  Hence the "TODO" in the README.
- I do have some design specifications when choosing the colors.  For example, red is used sparingly as it is supposed to resemble important, critical or dangerous elements.
- I am open to any suggestions regarding the overall design.
- I want to add more colors, I feel a bit limited sometimes.
